content/en/ecosystem/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: OpenTelemetry Ecosystem
+linkTitle: Ecosystem
+description: >-
+  OpenTelemetry's thriving ecosystem of official and community-contributed
+  components and examples
```

Comment:
based on my comment to also include page(s) on vendors, integrators, adopters this description might need to change a little bit.

Reply:
Done, PTAL